tests/scripts/perf-prep.sh

```diff
@@ -44,39 +44,35 @@ echo "branch = $perfBranch"
 echo "architecture = $perfArch"
 echo "configuration = $perfConfig"
 
-# Install nuget to download benchview package, which includes the script machinedata.py for machine data collection
-wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
-chmod u+x ./nuget.exe
-./nuget.exe install Microsoft.BenchView.JSONFormat -Source http://benchviewtestfeed.azurewebsites.net/nuget -OutputDirectory ./tests/scripts -Prerelease -ExcludeVersion
+# Since not all perf machines have Mono we cannot run nuget locally to get the Benchview tools
+# Instead we curl the package feed and use grep and sed to find the newest package.
+# We grep for content type and that returns us strings that contain the path to the nupkg
+# Then we match only the last line using '$' and use the s command to replace the entire line
+# with what we find inside of the quotes after src=.  We then jump to label x on a match and if 
+# we don't match we delete the line.  This returns just the address of the last nupkg to curl.
+curl "http://benchviewtestfeed.azurewebsites.net/nuget/FindPackagesById()?id='Microsoft.BenchView.JSONFormat'" | grep "content type" | sed "$ s/.*src=\"\(.*\)\".*/\1/;tx;d;:x" | xargs curl -o benchview.zip
```

Comment:
Here, too, I recommend changing the arguments to sed from  
`$ s/.*src=\"\(.*\)\".*/\1/;tx;d;:x`  
to  
`$ s/.*src=\"\([^\"]*\)\".*/\1/;tx;d;:x`  
It works for now, but this would make sure that the logic doesn't break if the XML changes form to include any props after the src="..." prop.